docs/manual/manual_telemetry.rst

```diff
@@ -0,0 +1,235 @@
+	| Vertiq 4006 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Supported Telemetry
+=====================
+The structure of the supported telemetry messages and how to trigger them varies between different protocols. The sections below cover these details for different protocols.
+
+Hobby Protocols
+****************
+ESC telemetry is available when using :ref:`Hobby Protocols <manual_hobby>` with Vertiq modules. Telemetry messages are sent from the module as a serial
+message at the baud rate configured on the module.
+
+The exact method for requesting telemetry varies depending on what type of hobby protocol is being used, but the structure and content of 
+the telemetry sent is the same across all hobby protocols.
+
+Structure and Content
+######################
+The structure and content of the ESC telemetry message available when using :ref:`Hobby Protocols <manual_hobby>` is based on the `telemetry
+of KISS and BLHeli ESCs <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_. 
+
+**There is one key difference between the telemetry sent by Vertiq modules and the KISS standard. The KISS standard specifies sending ERpm data in the telemetry message. ERpm data
+must be converted into RPM using the number of poles of the motor to be useful. Vertiq modules directly send RPM data, which eliminates the
+need to perform any conversion.**
+
+.. note:: If your flight controller expects to receive ERPM data from these telemetry messages, it may not interpret the RPM data sent by Vertiq modules correctly by default.
+    Additional configuration may be necessary on the flight controller to properly intepret the received RPM data as RPM and not as ERPM. Refer to your flight controller's 
+    documentation.
+
+The telemetry sent by Vertiq modules when using :ref:`Hobby Protocols <manual_hobby>` is specified in the table below. The message
+consists of 10 bytes:
+
+	+----------+-------------------------+-----------------------+
+	| **Byte** | **Content**             | **Units**             |

```

Comment:
are these all floats besides CRC? should we specify?

Reply:
What do you mean by a float? If you mean a 4 byte floating point number, then no, they are not floats. These encode these numbers as essentially integers. If you grab the voltage high byte and low byte, put them together, and interpret them as twos complement number, that tells you the voltage in centivolts. So the resolution is limited to the centivolt level. Same as the KISS ESC standard says. You can see the example code lower on this page or the linked KISS ESC standard for an in-depth demonstration of how this works